.github/workflows/tag.yaml

```diff
@@ -0,0 +1,42 @@
+name: Create Tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release tag'
+        required: true
+
+jobs:
+  tag:
+    name: Create tag
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
+      - name: 'Get previous tag'
+        id: previoustag
+        uses: "WyriHaximus/github-action-get-previous-tag@v1"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Note that you don't have a tag yet so this step will fail. Either create a 0.0.0 tag manually before the first release or add the `fallback: 0.0.0` parameter here.

Reply:
Thanks I created manually tag 0.0.0